### PR TITLE
Rwcarlsen upd release

### PR DESCRIPTION
**Pull request by Baaaaam**
push @rwcarlsen branch into the install_docs_reworks branch 

it still need to be updated with the last change (between RC3 and actual release)

### CODE REVIEW

**Comment by Baaaaam** [comment]
@mbmcgarry could you review this one:
it is just transferring the change into this branch allowing to merge everything in the same time

File: source/arche/testing.rst [hunk 1]
```diff
@@ -1,8 +1,8 @@
 
 .. _testing:
 
-Testing Agents and Modules
-==========================
+Testing and Debuging Archetypes
```
Comment (mbmcgarry) [inline source/arche/testing.rst]: Debugging

File: source/previous/v1.4.rst [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+==================================
+Cyclus 1.4.0 Release
+==================================
+:Author: Cyclus Developers
+:Website: http://fuelcycle.org/
+:Contact: https://groups.google.com/forum/#!forum/cyclus-dev
+
+The Cyclus core team is excited to announce the latest stable version of the
+cyclus ecosystem!
```
Comment (mbmcgarry) [inline source/previous/v1.4.rst]: Cyclus

File: source/previous/v1.4.rst [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+Major features
+==============
```
Comment (mbmcgarry) [inline source/previous/v1.4.rst]: Features

File: source/previous/v1.4.rst [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+* Fixed floating-point comparison bugs in DRE causing incorrectly failed
+  resource trades.
+
+* Determinism/reproducibility improved by calling agents' resource exchange
+  related functions in consistent order.
+
+* Fixed invalid memory access bugs affecting newer and mac compilers.
+
+* xinclude substitutions in input files are now procesed for the input file
```
Comment (mbmcgarry) [inline source/previous/v1.4.rst]: processed

File: source/user/dbdoc.rst [hunk 5]
```diff
@@ -81,6 +81,69 @@ tables.
...
+activating this table.  This table provides the material inventory of each
+agent at each time step subdivided by internal buffers (e.g.
+``cyclus::toolkit::ResBuf``).  There is one row for each agent on each time
+step for each sub-buffer for each nuclide.
+
+* **SimId** (uuid)
+
+* **AgentId** (int): ID of the agent holding the inventory 
+
+* **Time** (int): time step this inventory was in the given agent.
```
Comment (mbmcgarry) [inline source/user/dbdoc.rst]: Time step

**Comment by mbmcgarry** [comment]
Thanks @rwcarlsen and @Baaaaam !!

**Pull request merged**